heartbeat/monitors/util.go

```diff
@@ -342,3 +448,55 @@ func filterIPs(ips []net.IP, filt func(net.IP) bool) []net.IP {
 	}
 	return out
 }
+
+// MakeJobSetting creates a new JobSettings structure without any global event fields.
+func MakeJobSetting(name string) JobSettings {
+	return JobSettings{Name: name}
+}
+
+// WithFields adds new event fields to a Job. Existing fields will be
+// overwritten.
+// The fields map will be updated (no copy).
+func (s JobSettings) WithFields(m common.MapStr) JobSettings {
+	s.AddFields(m)
+	return s
+}
+
+// AddFields adds new event fields to a Job. Existing fields will be
+// overwritten.
+func (s *JobSettings) AddFields(m common.MapStr) { addFields(&s.Fields, m) }
+
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method HostJobSettings.AddFields should have comment or be unexported